recipes-devtools/luasocket/luasocket_3.0.bb

```diff
@@ -24,6 +24,7 @@ do_compile() {
 do_install() {
     cd ${S}/src
     oe_runmake install
+    oe_runmake install-unix
```

Comment:
As far as I see [here in code](https://github.com/diegonehab/luasocket/blob/5b18e475f38fcf28429b1cc4b17baee3b9793a62/src/makefile#L422) `install-unix` requires install, so maybe we can just change the line above to `install-unix` instead of having two?

Reply:
Seems good to me!